### PR TITLE
fix(core-blocks): magnetising table and rotary generator fixes

### PR DESCRIPTION
**Pull request by kamrantekkit**
fix(MagnestisingTable): Bug fix for duplication issue when Table's chunk is unloaded and loaded as well on server stop/start.

fix(Rotary-Generator): Improved code of the ChunkLoad Handler. As before was causing rapid chunk loading calls to the rotary gen's chunk

### CODE REVIEW

File: src/main/java/tech/brettsaunders/craftory/tech/power/core/block/machine/magnetiser/MagnetisingTable.java [hunk 1]
```diff
@@ -35,195 +39,216 @@
...
-      return;
-    }
-    if (event.getPlayer().isSneaking()) {
-      return;
+    @Override
+    public void beforeSaveUpdate() {
+        super.beforeSaveUpdate();
+        if (framePlaced) {
+            if (itemFrame == null && (!findFrame() || !framePlaced)) return;
+            System.out.println(frameItem);
```
Comment (brettsaunders21) [inline src/main/java/tech/brettsaunders/craftory/tech/power/core/block/machine/magnetiser/MagnetisingTable.java:94]: Please remove this debug line

**Review by brettsaunders21 — CHANGES_REQUESTED** [review]

**Comment by brettsaunders21** [comment]
Also version needs updating to v0.8.2

**Comment by kamrantekkit** [comment]
My bad, I corrected them now

**Review by brettsaunders21 — APPROVED** [review]
Looks good

**Pull request merged**